multicluster/apis/multicluster/v1alpha1/resourceexport_types.go

```diff
@@ -0,0 +1,87 @@
+/*
+Copyright 2021 Antrea Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"antrea.io/antrea/pkg/apis/crd/v1alpha2"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// ResourceExportSpec defines the desired state of ResourceExport
+type ResourceExportSpec struct {
+	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
+	// Important: Run "make" to regenerate code after modifying this file
+
+	// ClusterID specifies the member cluster this resource exported from.
+	ClusterID string `json:"clusterID,omitempty"`
+	// Name of exported resource.
+	Name string `json:"name,omitempty"`
+	// Namespace of exported resource.
+	Namespace string `json:"namespace,omitempty"`
+	// Kind of exported resource.
+	Kind string `json:"kind,omitempty"`
+
+	// If exported resource is Service.
+	Service *v1.ServiceSpec `json:"service,omitempty"`
```

Comment:
i thought we planned to export customized service spec (subset of ServiceSpec + additional Services related fields), similarly for others below

Reply:
what additional Service r u referring to?

Reply:
done.